language/libra-vm/src/libra_transaction_executor.rs

```diff
@@ -668,15 +676,30 @@ impl LibraVM {
                     if let Some(label) = counter_label {
                         USER_TRANSACTIONS_EXECUTED.with_label_values(&[label]).inc();
                     }
-                    (vm_status, output)
+                    (vm_status, output, Some(sender))
                 }
                 Ok(PreprocessedTransaction::WriteSet(txn)) => {
-                    self.process_writeset_transaction(data_cache, *txn, &log_context)?
+                    let (vm_status, output) =
+                        self.process_writeset_transaction(data_cache, *txn, &log_context)?;
+                    (vm_status, output, Some("write_set".to_string()))
+                }
+                Err(e) => {
+                    let (vm_status, output) = discard_error_vm_status(e);
+                    (vm_status, output, None)
```

Comment:
This should be a pure signature error. Should it be `None`?

Reply:
yes correct, that is what it looks like, just figured I'd bundle any error under None and print out a "malformed transaction" which for now we know to be only an invalid signature (and maybe forever?).
I somewhat did not want to be too explicit in the face of the `Err(e)`, I can change that if we think is important